geotrek/common/parsers.py

```diff
@@ -1918,6 +1932,42 @@ def normalize_field_name(self, name):
         return name
 
 
+class OpenStreetMapAttachmentsParserMixin(AttachmentParserMixin):
+    base_url_wikimedia = "https://api.wikimedia.org/core/v1/commons/file/"
+    non_fields = {"attachments": ("tags.wikimedia_commons", "tags.image")}
+    default_license_label = "CC-by-sa 4.0"
+
+    def filter_attachments(self, src, val):
+        attachments = []
+        wikimedia, image = val
+
+        if wikimedia and "File:" in wikimedia:
+            # Wikimedia Commons API url
+            filename = wikimedia.split(":")[1]
+            filename.replace(" ", "_")
+
+            url = self.base_url_wikimedia + filename
```

Comment:
ne concatene pas des chaines ede caractere

utilise les fstring ou les url format

```suggestion
            url =  f"{self.base_url_wikimedia}{filename}"
```